cumulus/client/consensus/aura/src/collators/mod.rs

```diff
@@ -662,6 +664,34 @@ mod tests {
 	}
 }
 
+/// Fetches relay chain storage proof requests from the parachain runtime.
+///
+/// Queries the runtime API to determine which relay chain storage keys
+/// (both top-level and child trie keys) should be included in the relay chain state proof.
+///
+/// Falls back to an empty request if the runtime API call fails or is not implemented.
+fn get_relay_proof_request<Block, Client>(
+	client: &Client,
+	parent_hash: Block::Hash,
+) -> RelayProofRequest
+where
+	Block: BlockT,
+	Client: ProvideRuntimeApi<Block>,
+	Client::Api: KeyToIncludeInRelayProof<Block>,
+{
+	client
+		.runtime_api()
+		.keys_to_prove(parent_hash)
+		.unwrap_or_else(|e| {
+			tracing::warn!(
+				target: crate::LOG_TARGET,
+				error = ?e,
+				"Failed to fetch relay proof requests from runtime, using empty request"
+			);
+			Default::default()
+		})
```

Comment:
```suggestion
		.unwrap_or_else(|e| {
			tracing::debug!(
				target: crate::LOG_TARGET,
				error = ?e,
				"Failed to fetch relay proof requests from runtime, using empty request"
			);
			Default::default()
		})
```

The API may not exists and then we clearly don't want to see these warnings :)